test/opt/opextinst_forward_ref_fixup_pass_test.cpp

```diff
@@ -0,0 +1,365 @@
+// Copyright (c) 2024 Google Inc.
```

Comment:
```suggestion
// Copyright (c) 2024 Google LLC
```

Before 2018, we were Google Inc. Since then, we are Google LLC